specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2020-10-01/examples/TablesUpdate.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "resourceGroupName": "oiautorest6685",
+    "workspaceName": "oiautorest6685",
+    "api-version": "2020-10-01",
+    "subscriptionId": "00000000-0000-0000-0000-00000000000",
+    "tableName": "table1",
+    "parameters": {
+      "properties": {
+        "retentionInDays": 40,
+        "isTroubleshootEnabled": true
```

Comment:
Can I omit any one of the properties in patch request?

Reply:
yes, it is possible to omit each of these properties.
is it possible to add another example for the same api?